src/mlpack/methods/ann/layer/gru_impl.hpp

```diff
@@ -76,6 +76,64 @@ GRU<InputDataType, OutputDataType>::GRU(
   gradIterator = outParameter.end();
 }
 
+// Copy Constructor definition
+template <typename InputDataType, typename OutputDataType>
+GRU<InputDataType, OutputDataType>::GRU(
+  const GRU& layer) : inSize(layer.inSize),
+  outSize(layer.outSize),
+  rho(layer.rho),
+  weights(layer.weights)
```

Comment:
```suggestion
GRU<InputDataType, OutputDataType>::GRU(const GRU& layer) :
    inSize(layer.inSize),
    outSize(layer.outSize),
    rho(layer.rho),
    weights(layer.weights)
```
I tend to be very picky about the style :)